modules/statistics/server/controllers/statistics.server.controller.js

```diff
@@ -187,6 +188,51 @@ exports.getPushRegistrationCount = function (callback) {
   });
 };
 
+/**
+ * Generate statistics based on the user last seen attribute
+ */
+exports.getLastSeenStatistic = function (type, callback) {
+  var now = moment();
+  var query = { };
+
+  switch (type) {
+    case 'never':
+      query.seen = { $exists: false };
```

Comment:
This will be a stable number (only very old profiles have this set to false) so we can drop this.